Index: b/Makefile
===================================================================
--- a/Makefile	2016-09-22 14:26:18.982969751 -0700
+++ b/Makefile	2016-09-22 14:27:01.510808635 -0700
@@ -11,17 +11,7 @@
 
 all: build
 
-build: build-arch build-indep
-
-# make man2html build available from project's base folder...
-build_man2html:
-	$(MAKE) -C x2goserver $@
-	$(MAKE) -C x2goserver-printing $@
-	$(MAKE) -C x2goserver-compat $@
-	$(MAKE) -C x2goserver-extensions $@
-	$(MAKE) -C x2goserver-xsession $@
-	$(MAKE) -C x2goserver-fmbindings $@
-	$(MAKE) -C x2goserver-pyhoca $@
+build: build-arch
 
 clean:
 	$(MAKE) -C x2goserver $@
@@ -36,15 +26,6 @@
 	$(MAKE) -C x2goserver $@
 	$(MAKE) -C x2goserver-printing $@
 	$(MAKE) -C x2goserver-compat $@
-	$(MAKE) -C x2goserver-extensions $@
-	$(MAKE) -C x2goserver-xsession $@
-	$(MAKE) -C x2goserver-fmbindings $@
-	$(MAKE) -C x2goserver-pyhoca $@
-
-build-indep:
-	$(MAKE) -C x2goserver $@
-	$(MAKE) -C x2goserver-printing $@
-	$(MAKE) -C x2goserver-compat $@
 	$(MAKE) -C x2goserver-extensions $@
 	$(MAKE) -C x2goserver-xsession $@
 	$(MAKE) -C x2goserver-fmbindings $@
Index: b/x2goserver-compat/Makefile
===================================================================
--- a/x2goserver-compat/Makefile	2016-09-22 14:26:18.982969751 -0700
+++ b/x2goserver-compat/Makefile	2016-09-22 14:27:01.510808635 -0700
@@ -26,27 +26,13 @@
 
 man_pages = `cd man && find * -type f`
 
-MAN2HTML_BIN  = man2html
-MAN2HTML_SRC  = man
-MAN2HTML_DEST = .build_man2html/html
-
 all: clean build
 
-build: build-indep
-
-build-indep: build_man2html
+build:
 
 build-arch:
 
-build_man2html:
-	mkdir -p $(MAN2HTML_DEST)
-	for man_page in $(man_pages); do mkdir -p `dirname $(MAN2HTML_DEST)/$$man_page`; done
-	for man_page in $(man_pages); do $(MAN2HTML_BIN) $(MAN2HTML_SRC)/$$man_page > $(MAN2HTML_DEST)/$$man_page.html; done
-
-clean: clean_man2html
-
-clean_man2html:
-	rm -rf `dirname $(MAN2HTML_DEST)`
+clean:
 
 install: install_scripts install_config install_man install_version
 
Index: b/x2goserver-extensions/Makefile
===================================================================
--- a/x2goserver-extensions/Makefile	2016-09-22 14:26:56.528710434 -0700
+++ b/x2goserver-extensions/Makefile	2016-09-22 14:27:01.510808635 -0700
@@ -23,29 +23,13 @@
 
 man_pages = `cd man && find * -type f`
 
-MAN2HTML_BIN  = $(shell which man2html)
-MAN2HTML_SRC  = man
-MAN2HTML_DEST = .build_man2html/html
-
 all: clean build
 
-build: build-arch build-indep
+build: build-arch
 
 build-arch:
 
-build-indep: build_man2html
-
-build_man2html:
-	if [ -n "$(MAN2HTML_BIN)" ]; then \
-	mkdir -p $(MAN2HTML_DEST); \
-	    for man_page in $(man_pages); do mkdir -p `dirname $(MAN2HTML_DEST)/$$man_page`; done; \
-	    for man_page in $(man_pages); do $(MAN2HTML_BIN) $(MAN2HTML_SRC)/$$man_page > $(MAN2HTML_DEST)/$$man_page.html; done; \
-	fi
-
-clean: clean_man2html
-
-clean_man2html:
-	rm -rf `dirname $(MAN2HTML_DEST)`
+clean:
 
 install: install_scripts install_config install_man install_version
 
Index: b/x2goserver-fmbindings/Makefile
===================================================================
--- a/x2goserver-fmbindings/Makefile	2016-09-22 14:26:18.982969751 -0700
+++ b/x2goserver-fmbindings/Makefile	2016-09-22 14:27:01.510808635 -0700
@@ -23,29 +23,13 @@
 
 man_pages = `cd man && find * -type f`
 
-MAN2HTML_BIN  = $(shell which man2html)
-MAN2HTML_SRC  = man
-MAN2HTML_DEST = .build_man2html/html
-
 all: clean build
 
-build: build-arch build-indep
+build: build-arch
 
 build-arch:
 
-build-indep: build_man2html
-
-build_man2html:
-	if [ -n "$(MAN2HTML_BIN)" ]; then \
-	    mkdir -p $(MAN2HTML_DEST); \
-	    for man_page in $(man_pages); do mkdir -p `dirname $(MAN2HTML_DEST)/$$man_page`; done; \
-	    for man_page in $(man_pages); do $(MAN2HTML_BIN) $(MAN2HTML_SRC)/$$man_page > $(MAN2HTML_DEST)/$$man_page.html; done; \
-	fi
-
-clean: clean_man2html
-
-clean_man2html:
-	rm -rf `dirname $(MAN2HTML_DEST)`
+clean:
 
 install: install_scripts install_config install_man install_version
 
Index: b/x2goserver-printing/Makefile
===================================================================
--- a/x2goserver-printing/Makefile	2016-09-22 14:26:18.982969751 -0700
+++ b/x2goserver-printing/Makefile	2016-09-22 14:27:01.510808635 -0700
@@ -24,29 +24,13 @@
 
 man_pages = `cd man && find * -type f`
 
-MAN2HTML_BIN  = $(shell which man2html)
-MAN2HTML_SRC  = man
-MAN2HTML_DEST = .build_man2html/html
-
 all: clean build
 
-build: build-arch build-indep
+build: build-arch
 
 build-arch:
 
-build-indep: build_man2html
-
-build_man2html:
-	if [ -n "$(MAN2HTML_BIN)" ]; then \
-	    mkdir -p $(MAN2HTML_DEST); \
-	    for man_page in $(man_pages); do mkdir -p `dirname $(MAN2HTML_DEST)/$$man_page`; done; \
-	    for man_page in $(man_pages); do $(MAN2HTML_BIN) $(MAN2HTML_SRC)/$$man_page > $(MAN2HTML_DEST)/$$man_page.html; done; \
-	fi
-
-clean: clean_man2html
-
-clean_man2html:
-	rm -rf `dirname $(MAN2HTML_DEST)`
+clean:
 
 install: install_scripts install_config install_man install_version
 
Index: b/x2goserver-xsession/Makefile
===================================================================
--- a/x2goserver-xsession/Makefile	2016-09-22 14:26:18.982969751 -0700
+++ b/x2goserver-xsession/Makefile	2016-09-22 14:27:01.511808655 -0700
@@ -30,17 +30,11 @@
 
 all: clean build
 
-build: build-arch build-indep
+build: build-arch
 
 build-arch:
 
-build-indep: build_man2html
-
-build_man2html:
-
-clean: clean_man2html
-
-clean_man2html:
+clean:
 
 install: install_scripts install_config install_man install_version
 
Index: b/x2goserver/Makefile
===================================================================
--- a/x2goserver/Makefile	2016-09-22 14:26:18.982969751 -0700
+++ b/x2goserver/Makefile	2016-09-22 14:27:01.511808655 -0700
@@ -30,36 +30,20 @@
 
 man_pages = `cd man && find * -type f`
 
-MAN2HTML_BIN  = $(shell which man2html)
-MAN2HTML_SRC  = man
-MAN2HTML_DEST = .build_man2html/html
-
 all: clean build
 
-build: build-arch build-indep
+build: build-arch
 
 build-arch: build_setgidwrappers
 
 build_setgidwrappers:
 	$(CC) $(CFLAGS) $(LDFLAGS) -DTRUSTED_BINARY=\"$(LIBDIR)/x2gosqlitewrapper.pl\" -o x2gosqlitewrapper x2gosqlitewrapper.c
 
-build-indep: build_man2html
-
-build_man2html:
-	if [ -n "$(MAN2HTML_BIN)" ]; then \
-	    mkdir -p $(MAN2HTML_DEST); \
-	    for man_page in $(man_pages); do mkdir -p `dirname $(MAN2HTML_DEST)/$$man_page`; done; \
-	    for man_page in $(man_pages); do $(MAN2HTML_BIN) $(MAN2HTML_SRC)/$$man_page > $(MAN2HTML_DEST)/$$man_page.html; done; \
-	fi
-
-clean: clean_arch clean_man2html
+clean: clean_arch
 
 clean_arch:
 	rm -f x2gosqlitewrapper
 
-clean_man2html:
-	rm -Rf `dirname $(MAN2HTML_DEST)`
-
 install: install_scripts install_config install_man install_version
 
 install_scripts:
